Source/santactl/Commands/SNTCommandMonitorMode.mm

```diff
@@ -53,8 +53,36 @@ + (NSString *)longHelpText {
   return [NSSet setWithArray:@[ @"mm" ]];
 }
 
+// Parse a time interval string into a number of minutes.
+// e.g. "10m" -> 10, "2h" -> 120, "3d" -> 3600
+- (NSTimeInterval)parseTimeInterval:(NSString *)duration {
+  NSScanner *scanner = [NSScanner scannerWithString:duration];
+  NSString *unit = nil;
+
+  NSInteger intValue = 0;
+  if ([scanner scanInteger:&intValue]) {
+    [scanner scanCharactersFromSet:[NSCharacterSet characterSetWithCharactersInString:@"smhd"]
+                        intoString:&unit];
+    if (unit == nil) {
+      return intValue;
+    }
+
+    if ([unit isEqualToString:@"s"]) {
+      // Round up and properly handle truncation: e.g. 61s -> 2 minutes
+      return (intValue + 59) / 60;
```

Comment:
I don't think we should support seconds. I think it'd be confusing when you request some duration and a longer duration is returned. If we really want to support seconds, we should change all the internal interfaces to support it - from santactl, to the daemon control controller, to the TMM class.